types/externs.d.ts

```diff
@@ -328,6 +329,9 @@ declare global {
       ph: 'B'|'b'|'D'|'E'|'e'|'F'|'I'|'M'|'N'|'n'|'O'|'R'|'S'|'T'|'X';
       s?: 't';
       id?: string;
+      id2?: {
```

Comment:
I can't believe this is actually a thing 😆

Reply:
id2 electric boogaloo